packages/office-ui-fabric-react/src/components/Persona/PersonaPresence.tsx

```diff
@@ -17,15 +17,24 @@ export class PersonaPresence extends React.Component<IPersonaProps, {}> {
   }
 
   public render(): JSX.Element | null {
-    let { presence } = this.props;
+    let { presence, coinSize } = this.props;
+    let coinSizeWithPresenceIconStyle = coinSize ? { fontSize: coinSize / 6, lineHeight: coinSize / 3 + 'px' } : undefined;
```

Comment:
Why is `px` only needed for line height and not the others?

Reply:
Confirmed what we talked about offline that the others are defaulting to add the 'px' while lineHeight is not.

Reply:
How does that behave in various browsers? 
Adding px explicitly makes it clearer what the intention is